index.js

```diff
@@ -25,66 +28,83 @@ function getRandomNumber(min, max) {
 function checkGuess() {
   // Get value from guess input element
   const guess = parseInt(guessInput.value, 10);
-  attempts = attempts + 1;
+  attempts++;
+  guessInput.value = "";
+  resetButton.style.display = "";
 
   hideAllMessages();
-
   if (guess === targetNumber) {
-    numberOfGuessesMessage.style.display = '';
-    numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
-
-    correctMessage.style.display = '';
-
+    maxGuessesMessage.innerHTML = `You made ${attempts} guesses`;
+    maxGuessesMessage.style.display = "";
+    correctMessage.style.display = "";
     submitButton.disabled = true;
     guessInput.disabled = true;
   }
 
   if (guess !== targetNumber) {
     if (guess < targetNumber) {
-      tooLowMessage.style.display = '';
+      if (guess <= 0) {
+        underZero.style.display = "";
+      } else {
+        tooLowMessage.style.display = "";
+      }
+    } else if (guess >= 100) {
+      overHundred.style.display = "";
     } else {
-      tooLowMessage.style.display = '';
+      tooHighMessage.style.display = "";
     }
 
     const remainingAttempts = maxNumberOfAttempts - attempts;
-
-    numberOfGuessesMessage.style.display = '';
-    numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guesses remaining`;
+    maxGuessesMessage.style.display = "";
+    if (attempts <= 5) {
+      maxGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guesses remaining`;
+    }
   }
 
-  if (attempts ==== maxNumberOfAttempts) {
+  if (attempts === maxNumberOfAttempts) {
+    for (let i = 0; i <= maxNumberOfAttempts.length; i++)
+      maxNumberOfAttempts[i].style.display = "";
     submitButton.disabled = true;
     guessInput.disabled = true;
   }
 
-  guessInput.value = '';
-
-  resetButton.style.display = '';
+  guessInput.value = "";
+  resetButton.style.display = "";
 }
 
 function hideAllMessages() {
-  for (let elementIndex = 0; elementIndex <= messages.length; elementIndex++) {
-    messages[elementIndex].style.display = 'none';
+  for (let i = 0; i < messages.length; i++) {
```

Comment:
ditto: the change in variable name makes it harder to see the one change that matters: `<=` to `<`.